server/utils/MailService.js

```diff
@@ -0,0 +1,94 @@
+import nodemailer from 'nodemailer';
+import path from 'path';
+import EventEmitter from 'events';
+import isString from 'lodash/isString';
+import isPlainObject from 'lodash/isPlainObject';
+import typeOf from 'typeof';
+
+class MailService extends EventEmitter {
+  constructor(props) {
+    super();
+
+    const {
+      transport = 'debug',
```

Comment:
I understand why you put these here, but please note `transport`, `from` and `templatePath` properties are required, i.e. the MailService should throw a `TypeError` if not provided. Here's a quick example

```javascript
import isString from 'lodash/isString';

if (!isString(props.from)) {
  throw new TypeError(`Invalid "from" property; expected string, received ${typeOf(props.from}`);
}
```

Reply:
as stated above. The checks and the defaults there are given in case someone doesn't provide a mail configuration as stated in the ticket. We can change this if needed.